client/webpack.config.js

```diff
@@ -0,0 +1,64 @@
+const path = require('path');
+const webpack = require('webpack');
+const ExtractTextPlugin = require("extract-text-webpack-plugin");
+
+module.exports = {
+    entry: [
+        'webpack-dev-server/client?http://localhost:8000', // WebpackDevServer host and port
```

Comment:
will it be bundled when we run in 'production' mode serving static bundle and not with dev server? Is webpack's socket connection and stuff going to be placed in bundle?